agents/schemas/profile.json

```diff
@@ -28,7 +28,7 @@
       "description": "Agent avatar image link"
     },
     "email" : {
-      "type": "string",
+      "type": "emails",
```

Comment:
hey @iainkirkpatrick, i don't think this is legit JSON Schema.

i'd expect `"type": "string"`

and then another line `"format": "email"`.

or am i missing some special `emails` type?

Reply:
ooooh my bad! good catch - just checked and i totally read the schema docs wrong! will fix with fix branch